src/cake.coffee

```diff
@@ -4,18 +4,19 @@ fs = require 'fs'
 glob = require 'glob'
 log = console.log
 
-module.exports = (task) ->
+baseDir = () ->
+  path.resolve(path.join(__dirname, '..'))
+
 
-  baseDir = path.resolve(path.join(__dirname, '..'))
-  binDir = path.join(baseDir, 'node_modules', '.bin')
+module.exports = (task) ->
 
   task 'build', ->
     run "rm -fr ./lib; ./node_modules/.bin/coffee -o lib -c src"
 
 
   task 'lint', ->
-    coffeeLintPath = path.resolve(path.join(baseDir, 'coffeelint.json'))
-    run "#{binDir}/coffeelint -f #{coffeeLintPath} src/*"
+    coffeeLintPath = path.resolve(path.join(baseDir(), 'coffeelint.json'))
```

Comment:
Don't you want `baseDir()` to always resolve to this module's root directory? I ask because then maybe you could get rid of the `baseDir()` function and inline that call in `run()`.